Automattic/wp-calypso
# Get rid of old plans code

Fixes #7056.

## Testing Instructions

1. Run Calypso with `make run`;
2. Make sure `/plans`, `/plans/compare` work (with `/plans/compare` redirecting to `/plans`);
3. Also, make sure that the NUX (signup) flow works, esp. the plans step.
4. Is there something I forgot to remove?

## Questions

2. Should we rename `PlanFeatures` to `Plans`? E.g. instead of `plansFeaturesList`, `PlanFeaturesMain` have just `Plans` like it was before we started working on plan features?

@gwwar @rralian @mtias @artpi.

Test live: https://calypso.live/?branch=remove/old-plans-code
